src/lib/asset-service/service/AssetService.ts

```diff
@@ -23,7 +23,7 @@ export class AssetService {
   }
 
   get assets(): Asset[] {
-    return Object.values(this.assetsById)
+    return Object.values(this.assetsById) as Asset[]
```

Comment:
q: It might be super stupid but, we could avoid casting this if assetsById was typed as `Record<AssetId, Asset>`